<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GÜVENLİ SÜRÜŞ SİSTEMİ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Sifre_Line">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>264</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Kullanici_Adi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>203</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kullanıcı Adı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sifre">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>264</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Şifre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Kullanici_Adi_Line">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>203</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Yeni_Kayit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>361</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Yeni Hesap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Giris">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Giriş</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>361</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Şifremi Unuttum</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
